plugins/osdn/ovs/bin/openshift-sdn-ovs

```diff
@@ -77,7 +77,7 @@ del_ovs_flows() {
 }
 
 add_subnet_route() {
-    source /etc/openshift-sdn/config.env
+    source /run/openshift/config.env
     local subnet_route="ip route add ${OPENSHIFT_CLUSTER_SUBNET} dev eth0 proto kernel scope link src $ipaddr"
```

Comment:
Later on, it will be nice if we could change this file to adopt pkg/{ipcmd, ovs} apis.